### PR TITLE
feat:  Intro to Node.js & NPM (#39515)

### PR DESCRIPTION
**Pull request by AryanSinghalGit**

### CODE REVIEW

**Comment by Rahul-Successive** [comment]
What is this debug log which you have pushed?

**Comment by AryanSinghalGit** [comment]
> What is this debug log which you have pushed?

It is generated by vscode may be due to use debugger.

File: extra/npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,49 @@
+0 info it worked if it ends with ok
```
Comment (Rahul-Successive) [inline extra/npm-debug.log]: please delete this file as well

File: extra/patterns/npm-debug.log [hunk 2]
```diff
@@ -0,0 +1,49 @@
+0 info it worked if it ends with ok
```
Comment (Rahul-Successive) [inline extra/patterns/npm-debug.log]: delete this one as well

**Pull request merged**